tests/test_models.py

```diff
@@ -1358,3 +1361,170 @@ def test_current_framework_agreement_with_countersigned_and_new_signed(self):
             )
             supplier_framework = self.get_supplier_framework()
             assert supplier_framework.current_framework_agreement.id == 5
+
+
+class TestDirectAwardProjects(BaseApplicationTest, FixtureMixin):
+    _direct_award_project_name = ''
+
+    def test_create_new_project(self):
+        with self.app.app_context():
+            self.setup_dummy_user(role='buyer')
+
+            project = DirectAwardProject(name=DIRECT_AWARD_PROJECT_NAME, users=User.query.all())
+            db.session.add(project)
+            db.session.commit()
+
+            assert project.id is not None
+            assert project.name == DIRECT_AWARD_PROJECT_NAME
+            assert isinstance(project.created_at, datetime)
+            assert project.completed_at is None
+            assert project.active is True
+
+    def test_serialize_project_contains_required_keys(self):
+        with self.app.app_context():
+            self.setup_dummy_user(role='buyer')
+
+            project = DirectAwardProject(name=DIRECT_AWARD_PROJECT_NAME, users=User.query.all())
+            db.session.add(project)
+            db.session.commit()
+
+            project_keys_set = set(project.serialize().keys())
+            assert {'id', 'name', 'created_at', 'completed_at', 'status', 'active'} <= project_keys_set
+
+    def test_create_new_project_creates_project_users(self):
+        with self.app.app_context():
+            self.setup_dummy_user(role='buyer')
+
+            assert len(DirectAwardProjectUser.query.all()) == 0
+
+            project = DirectAwardProject(name=DIRECT_AWARD_PROJECT_NAME, users=User.query.all())
+            db.session.add(project)
+            db.session.commit()
+
+            assert len(DirectAwardProjectUser.query.all()) == len(User.query.all())
+
+    def test_direct_award_project_statuses(self):
+        with self.app.app_context():
+            project = DirectAwardProject(name=DIRECT_AWARD_PROJECT_NAME, users=User.query.all())
+            db.session.add(project)
+            db.session.commit()
+            assert project.status == 'live'
+
+            project.active = False
+            db.session.add(project)
+            db.session.commit()
+            assert project.status == 'abandoned'
+
+            project.completed_at = datetime.utcnow()
+            db.session.add(project)
+            db.session.commit()
+            assert project.status == 'completed'
```

Comment:
I guess I'm understanding you wrong.

We currently have an `active` boolean, not null and a `completed_at` datetime nullable. Are you just saying that `completed_at` should be renamed? If so that's fine.

Status is derived from those two properties already; it doesn't exist in the DB itself..